### PR TITLE
ActiveRecord 5.1

### PR DESCRIPTION
**Pull request by glaszig**
just started a new project on rails 5.1.0.beta1 and i need rgeo to work.
first time for me to mock around with AR in any meaningful way -- sorry if i'm screwing things.
tests are green on my side with postgresql 9.5 on ubuntu 16.04.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ rvm:
   - 2.2.6
   - jruby-9.0.5.0
 gemfile:
-  - gemfiles/ar50.gemfile
+  - gemfiles/ar51.gemfile
```
Comment (matthewrudy) [inline .travis.yml]: this means that the 5.0 tests no longer run on travis

Comment (teeparham) [inline .travis.yml]: If ActiveRecord has changed a lot internally, I like this style (a major version bump to the adapter and maintaining separate git & release branches of the adapter, corresponding to AR 5.0 & 5.1). If it's a minor change, I'll support both AR versions in a single branch.

Comment (glaszig) [inline .travis.yml]: the main issue is that the signature of `type_to_sql` [changed to accept keyword arguments](https://github.com/rails/rails/commit/ae39b1a0#diff-a0775e1ec64264dc76a8ee71a5211ae3) and i'm not aware of a backward-compatible _and_ elegant approach to support both. i wouldn't want to use ruby version checks just to have cross-version support.

separate branches seem to be the way to go.

**Comment by serggl** [comment]
Would love to see this merged!

**Comment by glaszig** [comment]
anything i can or have to do to bring this forward? are we waiting for 5.1 final?

**Comment by scottfromsf** [comment]
I too would love to see this PR merged in.

**Comment by tszerszen** [comment]
Yeah, I'm waiting for this too!

**Comment by PelagicDev** [comment]
We're all waiting for this to be merged. :)

Since we're out of `beta` and now on `rc1`, things should be in a good place to have it merged.

**Comment by ecleel** [comment]
Rails 5.1 is out now. We'll be great if this PR merged.

**Comment by PelagicDev** [comment]
@teeparham, curious if we could get an explanation on why this isn't merged yet: is there process 
 that the Rgeo team follows that is still in progress, thus pending approval?

Just trying to get some sort of response for everyone who is following this PR. Thanks!

**Comment by teeparham** [comment]
For future reference, here are some example comments that would be helpful:

- Here's the Gemfile code to test this branch.
- I tested this on my site & it worked (on platform x, with ruby x, with rails x, ...).
- I would like to send you some beer for being a swell person.

"Why isn't this merged" is not helpful. "Rails 5.1 is out" is not helpful. "We're waiting for this 😺 " is not helpful.

I will have a release out today.

**Comment by PelagicDev** [comment]
Duly noted and completely fair @teeparham.

Only speaking for me here, but my assumption was that the tests for this Gem were covering the key areas. That said, I do understand that it's always helpful to have feedback of the PR actually running/working from someone's project.

Thanks for your time.

**Comment by fordfischer** [comment]
@teeparham I would like to send you a beer! Thank you for maintaining this library that literally lets me have a job.  DM me on twitter @savatexas and I'll give you Venmo or PayPal info so you can collect =)

**Comment by teeparham** [comment]
ActiveRecord 5.1 support is released as version 5.0.0 (on 5/1, no less 🎉 ).

https://rubygems.org/gems/activerecord-postgis-adapter

**Comment by MtnBiker** [comment]
Thank you to everyone for accomplishing this.

**Comment by teeparham** [comment]
Note: gem version 5.0.0 has been yanked. Version 5.0.1 is released.

See #254

**Pull request merged**